connector/src/main/java/com/google/cloud/spark/bigquery/v2/ArrowInputPartition.java

```diff
@@ -15,9 +15,7 @@
  */
 package com.google.cloud.spark.bigquery.v2;
 
-import com.google.cloud.bigquery.connector.common.BigQueryReadClientFactory;
-import com.google.cloud.bigquery.connector.common.ReadRowsHelper;
-import com.google.cloud.bigquery.connector.common.ReadSessionResponse;
+import com.google.cloud.bigquery.connector.common.*;
```

Comment:
Please avoid `.*` imports

Reply:
sorry about that, this is a fresh intellij install.  Is there a style file I should import?

Reply:
Not yet unfortunately... Once we will have the new architecture and the move to maven I will integrate a [google-java-format](https://github.com/google/google-java-format#third-party-integrations) maven plugin.